github-issue/check.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import requests
+import json
+import sys
+import os
+import datetime
+from pprint import pprint
+import dateutil.parser
+
+def msg(msg, *args, **kwargs):
+  if isinstance(msg, dict):
+    pprint(msg, stream=sys.stderr)
+  else:
+    print(msg.format(*args, **kwargs), file=sys.stderr)
+
+def get_newest_dates(lst, value):
+    for index, val in enumerate(lst):
+        if val >= value:
+            return lst[index:]
+    return [lst[-1]]
+
+def _check(instream):
+    payload = json.load(instream)
+    source = payload['source']
+
+    token = source['token']
+    repository = source['repository']
+    number = source['number']
```

Comment:
we want this resource to represent all matching issues. each `get` will give you only one issue, but we want the check to return a list of all matching issues so that we can find new ones without having an id hardcoded in the pipeline.

the way it works ([docs](https://concourse-ci.org/implementing-resource-types.html)) is that `check` will be passed a `version` which is the latest known version, and we want to return a list of all the new versions. if version is not passed i think we probably want to just return the most recent one, so that we don't trigger jobs for all historical issues